File: src/main/java/com/example/a71p/MainActivity.java
package com.example.a71p;

import android.content.Intent;
import androidx.appcompat.app.AppCompatActivity;
import android.os.Bundle;
import android.widget.Button;

public class MainActivity extends AppCompatActivity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        Button createNewAdvertButton = findViewById(R.id.createNewAdvert);
        createNewAdvertButton.setOnClickListener(view -> {
            startActivity(new Intent(this, CreateNew.class));
        });

        Button showAllItemsButton = findViewById(R.id.showAllItems);
        showAllItemsButton.setOnClickListener(view -> {
            startActivity(new Intent(this, ShowAllAdvert.class));
        });
    }
}

File: src/main/java/com/example/task71/MainActivity.kt
package com.example.task71

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        findViewById<Button>(R.id.createNewAdvert).setOnClickListener {
            startActivity(Intent(this,CreateNew::class.java))
        }

        findViewById<Button>(R.id.showAllItems).setOnClickListener {
            startActivity(Intent(this,ShowAllAdvert::class.java))
        }
    }
}